package.json

```diff
@@ -240,6 +240,10 @@
                 {
                     "when": "jlplotpaneFocus",
                     "command": "language-julia.plotpane-delete-all"
+                },
+                {
+                    "when": "editorLangId == julia",
+                    "command": "language-julia.clearAllInlineResultsInEditor"
```

Comment:
Should we add another condition whether there are actually any inline results?

Reply:
Hm, dunno. I'm not a fan of options/commands just not being there instead of them doing nothing.

Reply:
Could it be enabled/disabled, depending on whether there are inline results?

Reply:
No, I don't think so.